src/views/pages/student/results.ejs

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <%- include ("../../partials/student/head") %>
+    <link rel="stylesheet" href="/css/student/results.css">
```

Comment:
Ik mis nog een titel.
Hint: Check of de titel klopt met de pagina

Reply:
nu wel :)